test/runTest.ts

```diff
@@ -17,7 +17,8 @@ async function main(): Promise<void> {
             extensionTestsPath: path.resolve(repoRoot, 'dist', 'test', 'index'),
             extensionTestsEnv: {
                 DEBUGTELEMETRY: 'v'
-            }
+            },
+            version: "insiders"
```

Comment:
The tests aren't actually doing anything in this repo. Can you add something like this: https://github.com/microsoft/vscode-azurestaticwebapps/blob/master/test/global.test.ts#L37

Once you do that, I think you will see the test failing because you forgot this change:
https://github.com/microsoft/vscode-azurestorage/pull/728/files#diff-02aeed42126d9e2bd48df8e67b14fe98R22